css/index.css

```diff
@@ -112,6 +148,46 @@ p {
     justify-content: center;
 }
 
+.middle-content .boxes #one{
+    background: teal;
+}
+
+.middle-content .boxes #two{
+    background: gold;
+}
+
+.middle-content .boxes #three{
+    background: cadetblue;
+}
+
+.middle-content .boxes #four{
+    background: coral;
+}
+
+.middle-content .boxes #five{
+    background: crimson;
+}
+
+.middle-content .boxes #six{
+    background: forestgreen;
+}
+
+.middle-content .boxes #seven{
+    background: darkorchid;
+}
+
+.middle-content .boxes #eight{
+    background: hotpink;
+}
+
+.middle-content .boxes #nine{
+    background: indigo;
+}
+
+.middle-content .boxes #ten{
+    background: dodgerblue;
+}
+
```

Comment:
Excellent use of the id tag here is a great resource for an alternative css attribute that would allow you to do the same without having to write ids for each box. https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child